### PR TITLE
Feat/verifikasi request

### PR DESCRIPTION
**Pull request by rayfajars**
## OVERVIEW

1. add notes rejected

## PREVIEW

image

title: add notes rejected
project: Digiteam Inventaris Platform
participants: @rayfajars @doohanas @yoslie @Ibwedagama  @maulanayuseph @agunghide

### CODE REVIEW

File: src/components/RequestComponents/DetailRequest.vue [hunk 1]
```diff
@@ -166,38 +168,69 @@ export default {
...
+          title: "Ingin update status permohonan?",
+          text: "Permohonan akan diubah statusnya!",
+          icon: "warning",
+          showCancelButton: true,
+          confirmButtonColor: "#3085d6",
+          cancelButtonColor: "#d33",
+          confirmButtonText: "Kirim!",
+          cancelButtonText: "Batalkan",
+          input: "textarea",
+          inputPlaceholder: "Masukan detail catatan...",
```
Comment (yoslie) [inline src/components/RequestComponents/DetailRequest.vue:187]: same issue as before, it's `masukkan` (as a verb) not `masukan` (as a noun)

File: src/components/RequestComponents/DetailRequest.vue [hunk 2]
```diff
@@ -166,38 +168,69 @@ export default {
...
+              title: "Success",
+              text: "Berhasil update status permohonan",
+            })
+            .then(() => {
+              this.$store.dispatch("modals/close", this.name);
+              this.$emit("get-response-form");
+            });
+        })
+        .catch((err) => {
+          this.messageError = err.response.data.errors;
```
Comment (yoslie) [inline src/components/RequestComponents/DetailRequest.vue:223]: i can't find a purpose in `this.messageError`, is it used?

File: src/components/layouts/ActionTable.vue [hunk 3]
```diff
@@ -19,7 +19,10 @@ export default {
       );
     },
     btnVerifikasi() {
-      return this.status < statusObject.PENGAJUAN_SELESAI.value;
+      return (
+        this.status < statusObject.PENGAJUAN_SELESAI.value &&
+        this.status != statusObject.PENGAJUAN_DITOLAK.value
```
Comment (Ibwedagama) [inline src/components/layouts/ActionTable.vue:24]: minor:

hello @rayfajars just a minor thing, can you use a triple equal sign (===) for this value comparison?

File: src/components/RequestComponents/DetailRequest.vue [hunk 4]
```diff
@@ -71,6 +71,9 @@ export default {
         this.detailRequest.status > statusObject.PENGECEKAN_KELAYAKAN.value
       );
     },
+    rejectedRequest() {
+      return this.detailRequest.status == statusObject.PENGAJUAN_DITOLAK.value;
```
Comment (Ibwedagama) [inline src/components/RequestComponents/DetailRequest.vue:75]: minor:

hello @rayfajars just a minor thing, can you use a triple equal sign (===) for this value comparison?

**Review by doohanas — APPROVED** [review]

File: src/components/RequestComponents/DetailRequest.vue (hunk 1 above)
Comment (rayfajars) [reply src/components/RequestComponents/DetailRequest.vue]: sorry ,

i am fix [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/19/commits/77c51fd9ae9de350b3ee5171e19728daa3719f8b)

**Review by maulanayuseph — APPROVED** [review]

File: src/components/layouts/ActionTable.vue (hunk 3 above)
Comment (rayfajars) [reply src/components/layouts/ActionTable.vue:24]: i am fix  [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/19/commits/3886720fd9ffe8bbcbcb798343fd29c9126881b3)

File: src/components/RequestComponents/DetailRequest.vue (hunk 4 above)
Comment (rayfajars) [reply src/components/RequestComponents/DetailRequest.vue]: i am fix  [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/19/commits/3886720fd9ffe8bbcbcb798343fd29c9126881b3)

File: src/components/RequestComponents/DetailRequest.vue (hunk 2 above)
Comment (rayfajars) [reply src/components/RequestComponents/DetailRequest.vue:223]: this code for get response error message in field 

like 

image

i and used in this line

image

**Review by Ibwedagama — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

File: src/components/RequestComponents/DetailRequest.vue (hunk 2 above)
Comment (yoslie) [reply src/components/RequestComponents/DetailRequest.vue:223]: is it in DetailRequest.vue ?

Comment (yoslie) [reply src/components/RequestComponents/DetailRequest.vue:223]: image

I still can't find it initiated in data or props or binded #cmiiw

**Review by yoslie — APPROVED** [review]

**Pull request merged**